components/brave_wallet_ui/components/desktop/wallet-onboarding/import-meta-mask-or-legacy/index.tsx

```diff
@@ -9,29 +9,54 @@ import {
   MetaMaskIcon,
   BraveIcon,
   ArrowIcon,
-  LostButton
+  LostButton,
+  PasswordTitle,
+  CheckboxRow
 } from './style'
+import { Checkbox } from 'brave-ui'
 import { PasswordInput } from '../../../shared'
 import { NavButton } from '../../../extension'
 import locale from '../../../../constants/locale'
 
 export interface Props {
   onSubmit: () => void
   onPasswordChanged: (value: string) => void
+  onConfirmPasswordChanged: (value: string) => void
+  onImportPasswordChanged: (value: string) => void
   onClickLost: () => void
+  onCheckedPassword: (checked: boolean) => void
+  password: string
+  confirmedPassword: string
+  checkedPassword: boolean
```

Comment:
this naming is confusing, I thought it was checking if it meets password requirement when I first saw it.
It's better to rename it as `useSamePassword`

Reply:
Changed!